keyboards/keebio/viterbi/keymaps/met/keymap.c

```diff
@@ -0,0 +1,319 @@
+// Each layer gets a name for readability, which is then used in the keymap matrix below.
```

Comment:
I meant only Line 3 before. *This* line *is* required:

```suggestion
#include QMK_KEYBOARD_H

// Each layer gets a name for readability, which is then used in the keymap matrix below.
```

Reply:
whoopsie